internal/authintegration/integration_test.go

```diff
@@ -254,6 +262,9 @@ func (h *authHandler) OnPubKey(
 func (h *authHandler) OnAuthorization(
 	meta metadata.ConnectionAuthenticatedMetadata,
 ) (bool, metadata.ConnectionAuthenticatedMetadata, error) {
+	if meta.AuthenticatedUsername == "foo" {
```

Comment:
Not sure, but could it break the test?

Reply:
Now the tests passed, there are 2 known flaky tests, the forwarding agent one, and the auditlog one. It's one of the things that is still left to be investigated. I don't think it's relevant here (we are not touching the auditlog anyway).